app/services/whatsapp/providers/whatsapp_cloud_service.rb

```diff
@@ -23,13 +23,19 @@ def send_template(phone_number, template_info)
   end
 
   def sync_templates
-    response = HTTParty.get("#{business_account_path}/message_templates?access_token=#{whatsapp_channel.provider_config['api_key']}")
+    templates = fetch_wa_templates("#{business_account_path}/message_templates?access_token=#{whatsapp_channel.provider_config['api_key']}")
```

Comment:
```suggestion
    templates = fetch_whatsapp_templates("#{business_account_path}/message_templates?access_token=#{whatsapp_channel.provider_config['api_key']}")
```

it's better to use the full name, easier to search if someone new is trying to debug something